### ISSUE TITLE
Request: Reduce tool icon clutter on edit page

### ISSUE THREAD

**Issue by iisworks** [skogsaas/homey.dashboards#82]
Right now the edit page islooks a bit cluttered with all the tool icons on every widget.
Maybe show the edit icons/tools for a widget only when hovering over it?

**Comment by Torch1969** [comment]
Hovering doesn’t work (very well/at all) on tablets (at least on iPad), so please take that in consideration when judging this request :scream:

**Comment by iisworks** [comment]
Good point, maybe a single click on the widget to reveal the tool icons would be a more viable approach?
